src/audio/module_adapter/module/volume/volume.c

```diff
@@ -1294,7 +1294,9 @@ static int volume_prepare(struct processing_module *mod,
 
 #if CONFIG_IPC_MAJOR_4
 	cd->peak_cnt = 0;
+#if CONFIG_COMP_PEAK_VOL
 	cd->peak_report_cnt = CONFIG_PEAK_METER_UPDATE_PERIOD * 1000 / mod->dev->period;
+#endif
```

Comment:
#endif should goto line 1302?

Reply:
I'm not sure about this. Maybe all of peak_cnt reporting should go within the #if. But this is enough for now to fix the compilation error when the config isn't enabled

Reply:
I'd say 1302 needs to be included as well.

s/COM/COMP/ typo also in cmmit title